Energy_Consumption_Schema.json

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "array",
+  "items": {
+    "$ref": "#/definitions/WelcomeElement"
+  },
+  "definitions": {
+    "WelcomeElement": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "data_year": {
+          "type": "string",
```

Comment:
As long as you're adding a schema, you might as well add validation rules to it as well.  How long can a string be?  What's the maximum/minimum value for a number?